assembl/static2/js/app/components/home/video.jsx

```diff
@@ -3,6 +3,8 @@ import { connect } from 'react-redux';
 import { Translate } from 'react-redux-i18n';
 import { Grid, Row, Col } from 'react-bootstrap';
 
+import Medias from '../common/medias';
+
 class Video extends React.Component {
   constructor(props) {
```

Comment:
constructor not needed, you can do ```state = { isTextHigher: false }```

Reply:
It's an old file habibi. Not in the scope

Reply:
we try to refactor every file we touch to clean the codebase though :/ and this takes like 1 sec but as you wish, not important

Reply:
componentWillReceiveProps => getDerivedStateFromProps